.github/workflows/e2e.yml

```diff
@@ -66,14 +68,23 @@ jobs:
         run: |
           chmod 0600 $PGPASSFILE
           backend/db-setup/create-db.sh
+      - name: Set up nginx reverse proxy and application-local
+        run: |
+          sudo cp ./frontend/e2e/utils/localhost.simplereport.gov /etc/nginx/sites-available
```

Comment:
😮 does this actually work? We can just use a Github runner as a standard Linux system?

Reply:
Yup! The commands just run inside an ubuntu container, and has nginx installed by default